lib/manager/components/FeedFetchFrequency.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+import Icon from '@conveyal/woonerf/components/icon'
+import React from 'react'
+import {
+  DropdownButton,
+  MenuItem
+} from 'react-bootstrap'
+
+import {FREQUENCY_INTERVALS} from '../../common/constants'
+import {getComponentMessages} from '../../common/util/config'
+
+import type {FetchFrequency} from '../../types'
+
+type Props = {
+  fetchFrequency?: FetchFrequency,
+  fetchInterval?: number,
+  onSelectFetchFrequency: FetchFrequency => void,
+  onSelectFetchInterval: number => void
+}
+
+const FeedFetchFrequency = (props: Props) => {
+  const {onSelectFetchFrequency, onSelectFetchInterval} = props
+  let {fetchFrequency, fetchInterval} = props
+  // If frequency and interval are not defined, use default of one fetch per day.
+  if (!fetchFrequency) fetchFrequency = 'DAYS'
+  const intervals = FREQUENCY_INTERVALS[fetchFrequency]
+  if (!fetchInterval) fetchInterval = intervals[0]
```

Comment:
Could be shortened to:

```js
let {fetchFrequency = 'DAYS', fetchInterval = intervals[0]} = props
```

Reply:
What if they're null though? Is that possibility precluded with flow typing? I chose this approach because (if I'm not mistaken) if a `null` value is passed it will override the default value.